src/System Application/Test/Azure File Services API/src/AzureTestHelper.Codeunit.al

```diff
@@ -0,0 +1,82 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Test.Azure.Storage.Files;
+
+using System.Azure.Storage.Files;
+
+/// <summary>
+/// Provides common authorization functionality for using Azure storage accounts in tests for Azure Storage Services.
+/// </summary>
+codeunit 132516 "Azure Test Helper"
```

Comment:
Maybe something like 
AFS Init. Test Storage

?

Reply:
Yes, that sounds better. Updated.